README.md

```diff
@@ -21,9 +21,10 @@ For a quick introduction on Flex, [read our blog post](https://blog.newrelic.com
 
 ## Compatibility and requirements
 
-Flex requires a New Relic [Infrastructure Pro](https://newrelic.com/infrastructure/pricing) subscription or trial and is compatible with the following operating systems:
+Flex requires a New Relic [Infrastructure Pro](https://newrelic.com/infrastructure/pricing) subscription or trial and is compatible with the following operating systems/platforms:
 
 - Linux
+- Kubernetes
 - Windows (Experimental)
```

Comment:
Should the `(Experimental)` be removed from this line? the changelog reference official Windows support now.